utils/pr_slow_ci_models.py

```diff
@@ -64,7 +64,7 @@ def get_new_python_files_between_commits(base_commit: str, commits: List[str]) -
     return code_diff
 
 
-def get_new_python_files() -> List[str]:
+def get_new_python_files(diff_with_last_commit=False) -> List[str]:
```

Comment:
Use `diff_with_last_commit` to indiciate we are compare files in `main` (head vs parent)